batcher/aligned-batcher/src/lib.rs

```diff
@@ -544,56 +677,85 @@ impl Batcher {
             return None;
         }
 
-        // Set the batch posting flag to true
-        *batch_posting = true;
-
-        let current_batch_size = match cbor_serialize(&batch_verification_data) {
-            Ok(serialized) => serialized.len(),
-            Err(e) => {
-                error!(
-                    "Failed to serialize verification data: {:?}, resetting batch state",
-                    e
-                );
-                self.flush_queue_and_clear_nonce_cache().await;
+        let gas_price = match self.get_gas_price().await {
+            Some(price) => price,
+            None => {
+                error!("Failed to get gas price");
                 return None;
             }
         };
 
-        // check if the current batch needs to be splitted into smaller batches
-        if current_batch_size > self.max_batch_size {
-            info!("Batch max size exceded. Splitting current batch...");
-            let mut acc_batch_size = 0;
-            let mut finalized_batch_idx = 0;
-            for (idx, (verification_data, _, _, _)) in batch_state.batch_queue.iter().enumerate() {
-                acc_batch_size += match cbor_serialize(verification_data) {
-                    Ok(serialized) => serialized.len(),
+        // Multiply the gas price by 5 to allow for spike in gas price before submitting
+        let gas_price = match gas_price.checked_mul(U256::from(5)) {
```

Comment:
let's make this multiplier a constant so that it is more visible and easily modifiable

Reply:
added on f62da9ff5020d33a886493b8f2e3fc6287c75e67